karavan-web/karavan-app/src/main/webui/src/topology/TopologyToolbar.tsx

```diff
@@ -34,7 +34,7 @@ export function TopologyToolbar (props: Props) {
         <ToolbarContent>
             <ToolbarItem align={{default:"alignRight"}}>
                 <Tooltip content={"Add Integration Route"} position={"bottom"}>
-                    <Button className="dev-action-button" size="sm"
```

Comment:
Please keep classname

Reply:
I'm not sure why this changed. I didn't even open this file, let alone changed anything in it. Is it possible that the build I ran generated these changes? In any case I'll revert them manually.